### ISSUE TITLE
Is my training running?

### ISSUE THREAD

**Issue by Quambus** [jcjohnson/torch-rnn#189]
Hi everyone, I'm a complete noob with Python/Torch, but I thought this whole thing was really cool and decided to give it a try. I've managed to process my data and install Torch with all the other plugins. However, when I try train.lua and then sample.lua, its giving me nothing. My file name was a.h5 and a.json. Could anyone please help? Thanks in advance. 

![image](https://cloud.githubusercontent.com/assets/28588585/25886448/f2424ecc-35a0-11e7-8fff-2d2065448434.png)

**Comment by ChrisCummins** [comment]
Two things I'd check:

1. Is your system doing *something*? If you are using CUDA then you can check `nvidia-smi`, else use `top`.
1. Does it work if you pre-process and train on the included `data/tiny-shakespeare.txt` dataset?

**Comment by Quambus** [comment]
I'm not using CUDA, and I processed the Shakespeare data set, but when I try to sample it, its still not giving me anything. Thanks again

**Comment by ChrisCummins** [comment]
Did you train on Shakespeare before sampling?

**Comment by Quambus** [comment]
I tried, and this is what I'm getting 

![image](https://cloud.githubusercontent.com/assets/28588585/25887589/12fd9342-35a6-11e7-87f0-9904d1d81005.png)

**Comment by antihutka** [comment]
You're typing the commands into Torch's shell. You need to type them into your OS's shell.

**Comment by Quambus** [comment]
Ah, I'm doing some janky setup where I'm running Bash on Ubuntu since I have a PC. Could that be the problem?

**Comment by vintermann** [comment]
No, bash is the standard on ubuntu and I don't know what you would be running it on if you didn't have a PC. Just instead of writing "th" to start torch, you write "th train.lua -input_h5 " etc.

**Comment by Quambus** [comment]
Oh okay, thanks! I tried that and got this error:

![image](https://cloud.githubusercontent.com/assets/28588585/25902127/fe556722-35db-11e7-958f-4d54743f7eb8.png)

I thought I might be missing something so I tried to reinstall everything again but I still got this error. Thanks again for all the helpers

**Comment by rplaughlin** [comment]
Do `ls` and see which directory you have torch-rnn in (Probably "torch-rnn").
Then, just `cd torch-rnn` or whatever directory it was.
Now try training. `train.lua` is in the directory you just changed to.

Alternatively, you can append `torch-rnn/`(Or whichever directory it was) in front of `train.lua`, like this:
`th torch-rnn/train.lua -input_h5 tiny-shakespeare.h5 -input_json tiny-shakespeare.json`

Whichever method works best for you.
